internal/grpc/proto/scheduler_service.proto

```
@@ -0,0 +1,102 @@
+syntax = "proto3";
+
+package prism.scheduler.v1;
```

Buf (1.55.1)

Files with package "prism.scheduler.v1" must be within a directory "prism/scheduler/v1" relative to root but were in directory "internal/grpc/proto".

(PACKAGE_DIRECTORY_MATCH)